app/javascript/src/components/Dashboard/Contacts/index.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect } from "react";
+
+import EmptyListImage from "images/EmptyList";
+import { Button } from "neetoui";
+import { Container, Header } from "neetoui/layouts";
+
+import EmptyState from "components/Common/EmptyState";
+import SideMenuBar from "components/Common/SideMenuBar";
```

Comment:
Add an index.js within `Common` so that you reduce this statement into one single import statement.